src/dmd/link.d

```diff
@@ -34,6 +35,42 @@ version (Posix) extern (C) int pipe(int*);
 version (Windows) extern (C) int spawnlp(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnl(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnv(int, const char*, const char**);
+version (Posix) extern (C) extern __gshared char** environ;
+static if (__VERSION__ >= 2083)
```

Comment:
This is not in the `version (Posix)` so it'll fail on Windows, won't it ?

Reply:
Oh god I'm an idiot. Yeah, hang on.

Reply:
fixed. bleh, thanks